kuttl-test.yaml

```diff
@@ -23,7 +23,7 @@ commands:
   - command: kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/latest/release.yaml
     namespaced: false
   # wait for webhooks to be ready
-  - command: sleep 180
+  - command: sleep 240
```

Comment:
Looks scary. Do we really need to wait so long?

Reply:
We had failed e2e tests with 180 timeouts because the webhook pods were not ready.

Reply:
Timeout reverted.